charts/actions-runner-controller/templates/githubwebhook.ingress.yaml

```diff
@@ -1,10 +1,12 @@
 {{- if .Values.githubWebhookServer.ingress.enabled -}}
 {{- $fullName := include "actions-runner-controller-github-webhook-server.fullname" . -}}
 {{- $svcPort := (index .Values.githubWebhookServer.service.ports 0).port -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
 apiVersion: networking.k8s.io/v1beta1
-{{- else -}}
+{{- else if .Capabilities.APIVersions.Has "extensions/v1beta1" }}
```

Comment:
For kubernetes 1.19, both `extensions/v1beta1` and `networking.k8s.io/v1` coexist so this approach in fact breaks this helm chart since we end up with `apiVersion: extensions/v1beta1` here but also hit the `{{- if $.Capabilities.APIVersions.Has "networking.k8s.io/v1" }}` conditional below producing the following error:

```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "service" in io.k8s.api.networking.v1beta1.IngressBackend
helm.go:88: [debug] error validating "": error validating data: ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "service" in io.k8s.api.networking.v1beta1.IngressBackend
```

Reply:
How do you suppose this should be fixed?

Reply:
Maybe test for `"networking.k8s.io/v1"` explicitly here since we test explicitly below?

Reply:
Could someone raise an issue for this please with the details so this can be fixed

Reply:
Done: https://github.com/actions-runner-controller/actions-runner-controller/issues/1037